backend/Onied/Support/Support/Profiles/AppMappingProfile.cs

```diff
@@ -1,10 +1,38 @@
 using AutoMapper;
+using Support.Data.Models;
+using Support.Dtos.Chat.GetChat.Response;
+using Support.Dtos.Support.GetChats.Response;
 
 namespace Support.Profiles;
 
 public class AppMappingProfile : Profile
 {
     public AppMappingProfile()
     {
+        // GetChat
+        CreateMap<Chat, GetChatResponseDto>()
+            .ForMember(dest => dest.SupportNumber,
+                opt => opt.MapFrom(src => src.Support == null ? (int?)null : src.Support.Number));
+        CreateMap<Message, GetChatMessageItem>()
+            .ForMember(dest => dest.SupportNumber,
+                opt => opt.MapFrom(src => src.Chat.Support == null ? (int?)null : src.Chat.Support.Number))
```

Comment:
Во всех сообщениях supportNumber будет зависеть от привязанного сотрудника. Это неверно. Здесь исправление, которое поставит number только тогда, когда отправитель - текущий оператор.

```suggestion
            .ForMember(dest => dest.SupportNumber,
                opt => opt.MapFrom(src =>
                    src.Chat.Support == null ? (int?)null :
                    src.Chat.SupportId == src.UserId ? src.Chat.Support.Number : null))
```

Это примерно то, что я сделал в своих маппингах в своем pr. Но есть проблема: Это тоже неверно. Старые операторы будут отображаться как null.

Скорее всего, самым простым решением будет сохранять номер оператора в БД, если прямо соответствовать требованиям. Я не учел этого когда писал их.